### DISCUSSION TITLE
Binance - SysErrOutOfGas

### DISCUSSION THREAD

**Discussion by mdmudassir0143** [filecoin-project/community#613] [Developers]
I was transferring FIL got this status and not recieved the FIL on mainnet
trx id- bafy2bzacecpeuhnyerycofcnzmzqfv2nly7igwusczhbusr5pizvgck22utmg
is there any solution to recover?

**Comment by TippyFlitsUK** [comment]
Where did you transfer from @mdmudassir0143?

**Reply by mdmudassir0143** [reply]
I was transferring from binance to gate.io

**Reply by mdmudassir0143** [reply]
Actually I needed the Fil on chain to perform some trxns but binance was not supporting the Fil address so I thought first send to gate.io then on FIL chain

**Comment by jennijuju** [comment]
Is this an exchange?

**Comment by oogabooga12352342312** [comment]
same problem here.  sent from binance to ledger live. says that I have received it, but the balance isn't updating.

**Reply by mdmudassir0143** [reply]
You can contact with binance customer support 
There were some problems on their end
And in my case they rejected the previous transaction and deposited the funds in my account, issue reseolved.

**Comment by jennijuju** [comment]
We are aware of the deposit issue with binance and have been working with them since last Thursday to resolve it. Hopefully - their service will be back soon. In the meantime, please contact binance customer service to escalate!

**Reply by oogabooga12352342312** [reply]
Thank you, this is much appreciated

**Reply by jennijuju** [reply]
https://twitter.com/_jennijuju/status/1637607134027304961?s=46&t=-cjzlHWSvTETjubLtXL_ww

**Reply by mdmudassir0143** [reply]
Can we recover our funds?

**Comment by nesg-9424** [comment]
Same thing happened to me when I transfer from Binance.US to a cold wallet. Did anyone get this resolved?

**Reply by jennijuju** [reply]
Binance tech team informed us they have fixed it -  can you please contact their customer support and DM the case id on slack, i will double confirm with the team.

**Reply by nesg-9424** [reply]
I have created a support case on Binance, but no reply yet. Case ID #75944726

**Reply by jennijuju** [reply]
thank you - i willl reach out to the tech team

**Reply by oogabooga12352342312** [reply]
Haven't been able to do it, tried it again this morning. Customer support said they fixed it but they haven't

**Reply by nesg-9424** [reply]
> thank you - i willl reach out to the tech team

thank you for your help

**Reply by nesg-9424** [reply]
> 

I did not get any response from Binance yet.

**Reply by mdmudassir0143** [reply]
You can contact the binance customer support and if not get any response then share your cid to binance support twitter
https://twitter.com/_jennijuju/status/1637607134027304961?s=46&t=-cjzlHWSvTETjubLtXL_ww

**Reply by jennijuju** [reply]
Binance tech suggests that the issue has been resolved since last week, if you have any failed messages thats more recent, please post your message CID here (link to filfox works too)

**Reply by oogabooga12352342312** [reply]
Here's mine for reference
bafy2bzaceb5a64a432u4zpc57h5rupenodry2wu6nvwelnscr4ejojckris44

**Reply by jennijuju** [reply]
> I have created a support case on Binance, but no reply yet. Case ID #75944726

@nesg-9424 Binance suggested that the case was resolved

**Comment by jennijuju** [comment]
The Binance Tech team has suggested their integration is working, and asks users to contact  Binance US customer support for any furthur issues.

**Comment by czak22** [comment]
I sent my filecoins from Binance Erc-20-bfil to metamask Erc-20. The funds are invisible on any wallet. Is there any chance to solve my problem?

**Comment by drraghavendra** [comment]
The Binance Tech team has suggested their integration is working, and asks users to contact Binance US customer support for any furthur issues.

**Comment by jennijuju** [comment]
hopefully last update - apparently Binance & and its US entity has a separate deployment pipeline, so Binance US was lagging a bit. tho as of March 30th, patch deployment are shipped in both
